src/com/ableton/Pyenv.groovy

```diff
@@ -19,7 +41,10 @@ class Pyenv implements Serializable {
    * @param pyenvRoot Path to the installation of pyenv.
    * @return New instance of VirtualEnv object.
    */
-  static VirtualEnv createVirtualEnv(Object script, String python, String pyenvRoot) {
```

Comment:
In 3c18137f8ffb40f97e933e8237645ebc68c7e896, the docstrings for the parameters need to be removed.

Reply:
I'll take care of this during rebase.